base_classes/NXdata.nxdl.xml

```diff
@@ -309,6 +309,20 @@
 
 		</doc>
 	</attribute>
+	<attribute name="target">
+        <doc>
+			Points to the path of a field defining the data to which the `DATA` group refers.
+
+			This attribute indicates the origin of the data, thereby providing an association to
+			further metadata that define the physical quantity that the data represents.
+
+			Example:
+			    If the data corresponds to a readout of a detector, ``@target`` points
+				to that detectors data:
+
+				  @target: '/entry/instrument/detector/data' for a detector
```

Comment:
Typo: detectors --> detector's

Also, use a different example that doesn't use detector twice.